### PR TITLE
Tweaked ScalaDoc page text to 3.0.5

### PR DESCRIPTION
**Pull request by plinlor**

### CODE REVIEW

File: app/views/scaladoc.scala.html [hunk 1]
```diff
@@ -23,7 +23,7 @@
 <h1>ScalaTest's Scaladoc documentation</h1>
 
 <ul>
-<li>Scaladoc for 3.0.1 - 3.0.4 is coming soon. Please use the Scaladoc for 3.0.0 for these versions in the meantime.</li>
+<li>Scaladoc for 3.0.1 - 3.0.5 is coming soon. Please use the Scaladoc for 3.0.0 for these versions in the meantime.</li>
```
Comment (hosamaly) [inline app/views/scaladoc.scala.html]: The Scaladoc for 3.0.1 is published. Perhaps this line should be updated to reflect this (and the following line uncommented).

**Comment by plinlor** [comment]
@cla-bot[bot] check

**Comment by artimasites** [comment]
@cla-bot[bot] check

**Comment by bvenners** [comment]
This one is no longer relevant. Closing.

**Pull request closed without merging**